src/staticanalysis/bot/static_analysis_bot/clang/format.py

```diff
@@ -63,6 +83,7 @@ def run(self, revision):
             os.path.join(settings.repo_dir, path).encode('utf-8')  # needed for hglib
             for path in filter(settings.is_allowed_path, revision.files)
         ]
+
```

Comment:
Nit: unrelated whitespace change

Reply:
I added the whitespace in order to better separated, I don't want to mix the contexts.

Reply:
I like it too, it's just that it's unrelated to the current patch (anyway it's a "nit", you can ignore it).